Doc/library/re.rst

```diff
@@ -537,26 +544,39 @@ form.
 .. data:: DEBUG
 
    Display debug information about compiled expression.
+   Doesn't have correcsponding inline flag.
 
 
 .. data:: I
           IGNORECASE
 
    Perform case-insensitive matching; expressions like ``[A-Z]`` will also
-   match lowercase letters.  The current locale does not change the effect of
-   this flag.  Full Unicode matching (such as ``Ü`` matching ``ü``) also
-   works unless the :const:`re.ASCII` flag is also used to disable non-ASCII
-   matches.
-
+   match lowercase letters.  Full Unicode matching (such as ``Ü`` matching
+   ``ü``) also works unless the :const:`re.ASCII` flag is also used to disable
+   non-ASCII matches.  The current locale does not change the effect of this
+   flag unless the :const:`re.LOCALE` flag is also used.
+   Correcsponds the inline flag ``(?i)``.
+
+   Note that in Unicode patterns the expression ``[A-Z]`` will match not
+   only letters 'A' to 'Z' and 'a' to 'z', but will also match letters 'İ'
+   (U+0130, Latin capital letter I with dot above), 'ı' (U+0131, Latin small
+   letter dotless i), 'ſ' (U+017f, Latin small letter long s) and 'K' (U+212a,
+   Kelvin sign).  If the :const:`ASCII` flag is used, only letters 'a' to 'z'
```

Comment:
This is duplicated from above, so the same comments applied (maybe it shouldn't be duplicated?).

Reply:
Could you suggest how to avoid duplication?

Reply:
I didn't notice the duplicated part was on two separate files, so it's probably ok to leave it.